forem/forem
# Flaky test fix: commentModerationTools.spec.js

## What type of PR is this? (check all applicable)



## Description

Flaky test fix.

Okay, so the tests were failing because there were few requests which were not finishing on time before executing the tests and because of which the test-based requests timed-out. 

To fix this @maestromac added a code to intercept and wait for all requests which should get executed before the test start. The reference used for this fix was: https://filiphric.com/how-to-wait-for-page-to-load-in-cypress

## Related Tickets & Documents



- Closes #19667

## QA Instructions, Screenshots, Recordings

Just run the `commentModerationTools ` cypress test locally multiple times to make sure that they do pass.

## Added/updated tests?
